config/boards/matebook-e-go.conf

```diff
@@ -0,0 +1,185 @@
+# Qualcomm Snapdragon 8cx Gen 3 Adreno 690 Qualcomm WCN685x Wi-Fi 6E Qualcomm Snapdragon X55 5G Bluetooth 5.1
+declare -g BOARD_NAME="HUAWEI MateBook E Go"
+declare -g BOARDFAMILY="uefi-arm64"
+declare -g BOARD_MAINTAINER="chenxuecong"
+declare -g KERNEL_TARGET="gaokun"
+declare -g BRANCH="gaokun"
```

Comment:
What does this branch name "gaokun" mean?

Reply:
"Gaokun" is the code name for MateBook E Go.